app-i18n/ibus-input-pad/ibus-input-pad-1.4.99_pre20140916.ebuild

```diff
@@ -13,7 +13,6 @@ SRC_URI="https://github.com/fujiwarat/${PN}/releases/download/${MY_PV}/${MY_P}.t
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 RDEPEND="app-i18n/ibus
```

Comment:
The DEPENDs in this ebuild apart from RDEPEND are actually BDEPEND.

Reply:
Got it, I'll make that edit.